core/src/main/java/org/apache/accumulo/core/util/HostAndPort.java

```diff
@@ -105,7 +105,7 @@ public boolean hasPort() {
    *           occurring.
    */
   public int getPort() {
-    checkState(hasPort());
+    checkState(hasPort(), "The given address does not include a port");
```

Comment:
Just a minor wording suggestion:

```suggestion
    checkState(hasPort(), "the address does not include a port");
```

The reason for this is that the exception could also occur when the user has not "given" any address. For example, if it's our own internal code. In that case, the word "given" might be confusing.